astacus/coordinator/plugins/clickhouse/object_storage.py

```diff
@@ -91,12 +91,24 @@ def _copy_items_between(
         target_storage: BaseTransfer[Any],
         stats: StatsClient | None,
     ) -> None:
+        total_keys = len(keys)
         match self.copy_config:
             case DirectCopyConfig():
-                logger.info("Copying the keys using the cloud APIs")
-                target_storage.copy_files_from(source=source_storage, keys=keys)
+                logger.info("Copying %r keys using the cloud APIs", total_keys)
```

Comment:
The correct string formatting keyword here is `%i` for integers.. sorry I didn't spot this earlier

Reply:
Adressed. 
Looking into our code base, we have not used once `%i` for integers.
I read that `repr` is the optimal way to debug values. However since its definitely `int`, this should be fine 👍